build/gc-scenarios.yml

```diff
@@ -14,8 +14,38 @@ parameters:
   type: object
   default:
 
-  - displayName: GC - GcPerfSim
-    arguments: '--scenario gcperfsim --application.environmentVariables DOTNET_gcServer=1 --application.environmentVariables \"DOTNET_GCHeapCount={{ cores | minus: 2 }}\" --application.variables.tc 52 --application.variables.tagb 540 --application.variables.tlgb 0 --application.variables.lohar 0 --application.variables.pohar 0 --application.variables.sohsr 100-4000 --application.variables.lohsr 102400-204800 --application.variables.pohsr 100-204800 --application.variables.sohsi 0 --application.variables.lohsi 0 --application.variables.pohsi 0 --application.variables.sohpi 0 --application.variables.lohpi 0 --application.variables.sohfi 0 --application.variables.lohfi 0 --application.variables.pohfi 0 --application.variables.allocType reference --application.variables.testKind time --property scenario=gcperfsim --property pohfi=0'
+  # Server Runs.
+  - displayName: 0gb Server
+    arguments: '--scenario 0gb --application.environmentVariables DOTNET_gcServer=1 --application.environmentVariables \"DOTNET_GCHeapCount={{ cores | minus: 2 }}\" --property scenario=0gb'
+    condition: 'true'
+
+  - displayName: 2gb Server
+    arguments: '--scenario 2gb --application.environmentVariables DOTNET_gcWorkstation=1 --application.environmentVariables \"DOTNET_GCHeapCount={{ cores | minus: 2 }}\" --property scenario=2gb'
+    condition: 'true'
+
+  - displayName: 2gb Pinning Server
+    arguments: '--scenario 2gb --application.environmentVariables DOTNET_gcServer=1 --application.environmentVariables \"DOTNET_GCHeapCount={{ cores | minus: 2 }}\" --property scenario=2gb-pinning'
+    condition: 'true'
+
+  - displayName: 20gb Server
+    arguments: '--scenario 20gb --application.environmentVariables DOTNET_gcServer=1 --application.environmentVariables \"DOTNET_GCHeapCount={{ cores | minus: 2 }}\" --property scenario=20gb'
+    condition: 'true'
+
+  # Workstation Runs.
+  - displayName: 0gb Workstation
+    arguments: '--scenario 0gb --application.environmentVariables DOTNET_gcServer=0 --property scenario=0gb --application.variables.tc 2'

```

Comment:
The `--scenario 0gb` argument is to tell crank which scenario from the config file to run.
The `--property scenario=0gb` is to store a `{ "scenario": "0gb" }` element in the database, that we use to "display" the result for this line in POwerBI.

You can have multiple runs that use `--scenario 0gb` and then add some variation (like GC server/Workstation) but you can have multiple runs that have the same set of custom properties. In this case you have multiple runs with `--property scenario=0gb` and no other properties to differentiate them. How do you want PowerBI to show each of them separately then?

There are several options: 
- Use a distinct property named scenario, like `0gb-server`, `0gb-workstation`, ... and these will appear on the list of available scenario properties on the right.
- Add another property named `gc=workstation` or `gc=server`, and then change powerbi to show the list of available scenario properties, and also a list of available gc properties. The risk is that maybe you will add scenarios that don't have a gc property, and the UI will hide some of them without users understanding why if they don't know that a specific scenario has no gc property